src/ResourceManagement/Network/LoadBalancerFrontendImpl.cs

```diff
@@ -157,7 +157,7 @@ internal LoadBalancerFrontendImpl WithExistingPublicIPAddress (IPublicIPAddress
         ///GENMHASH:DD83F863BB3E548AA6773EF2F2FDD700:488D62DEC18443445C34B6C753A0435A
         internal LoadBalancerFrontendImpl WithExistingPublicIPAddress (string resourceId)
```

Comment:
C# withExistingSubnet(..) with-er creates instance of SubnetInner https://github.com/Azure/azure-sdk-for-net/blob/Fluent/src/ResourceManagement/Network/LoadBalancerFrontendImpl.cs#L144 but in Java it is SubResource() https://github.com/Azure/azure-sdk-for-java/blob/master/azure-mgmt-network/src/main/java/com/microsoft/azure/management/network/implementation/LoadBalancerFrontendImpl.java#L153